tests/lib/testlib/constants.py

```diff
@@ -52,3 +54,11 @@
 
 PORT1_IFACE_NAME = "port1"
 PORT2_IFACE_NAME = "port2"
+
+
+def get_test_ipv4_addrs():
```

Comment:
I am not clear why you need the private module variable at all.
IMO you can also keep the naming convention the same as the other constants here which are, i.e. `ipv4_addresses()`. 

```
def ipv4_addresses():
    return [...]
```

Just an opinionated suggestion.